sphinx/addnodes.py

```diff
@@ -247,6 +247,10 @@ def astext(self):
         return f'({super().astext()})'
 
 
+class desc_parameterline(nodes.General, nodes.Element):
+    """Node for a parameter line."""
```

Comment:
Please expand on this description, I can forsee forgetting this PR in the future and being confused as to what a "parameter line" is.